src/app/collection-page/collection-page.component.html

```diff
@@ -1,78 +1,58 @@
 <div class="container">
-    <div class="collection-page"
-         *ngVar="(collectionRD$ | async) as collectionRD">
-              <div *ngIf="collectionRD?.hasSucceeded" @fadeInOut>
-                      <div *ngIf="collectionRD?.payload as collection">
-              <ds-view-tracker [object]="collection"></ds-view-tracker>
-              <div class="d-flex flex-row border-bottom mb-4 pb-4">
-                <header class="comcol-header mr-auto">
-                  <!-- Collection Name -->
-                  <ds-comcol-page-header
-                          [name]="dsoNameService.getName(collection)">
-                  </ds-comcol-page-header>
-                  <!-- Collection logo -->
-                  <ds-comcol-page-logo *ngIf="logoRD$"
-                                   [logo]="(logoRD$ | async)?.payload"
-                                   [alternateText]="'Collection Logo'">
-                  </ds-comcol-page-logo>
+        <div class="collection-page" *ngVar="(collectionRD$ | async) as collectionRD">
```

Comment:
Would it be possible to undo all the changes to this file?  It appears that you may have just realigned the HTML, but it'd be best to revert all changes, otherwise you may cause code conflicts with other PRs.

Reply:
@GauravD2t : The changes to the `collection-page.component.html` are still not undone. They still are visible in the files changed tab in GitHub: https://github.com/DSpace/dspace-angular/pull/2275/files